build.sbt

```diff
@@ -10,7 +10,7 @@ scalacOptions ++= List(
   "-unchecked",
   "-deprecation",
   "-language:_",
-  "-target:jvm-1.7",
+  "-target:jvm-1.6",
```

Comment:
It's only the version for this plugin, so it can be loaded by an sbt in a Java 6 JVM. You need to be able to load the plugin even if you don't run the JMH tests there.